docs/changelog.md

```diff
@@ -8,6 +8,21 @@ toc:
 
 <!-- do-not-remove -->
 
+## 1.1.0 (2023-09-12)
+
+### Minor Changes
+
+- Added join support for OAS 3.1 definitions.
```

Comment:
Isn't this a feature?

Reply:
Yes, it is. So it goes as a minor change according to semver: https://semver.org/#summary

Reply:
> We're following https://keepachangelog.com/en/1.1.0/ format though. 

We are not following it from what I can see. Prior 1.0.0 we were using `Features` and `Fixes`.

![image](https://github.com/Redocly/redocly-cli/assets/3975738/1364f99d-1249-473e-98b2-8e5dc5c00ee3)


It was changed on 1.0.0 when we moved to to autogenerated changelog. I think we can adjust our script to keep `Features` and `Fixes` instead of `Minor Changes` and `Patch Changes`.

@adamaltman @tatomyr what do you think?

Reply:
I believe the current format is widely used by comunity, so it's better to adhere to it as it is. 
I think I can add an admonition to the changelog that we've adopted the Changesets approach since v1.0 for the time being. And later we can discuss what to do with the changelog format.

Reply:
> I believe the current format is widely used by comunity

Other formats are also used by community, see examples:

https://github.com/stoplightio/spectral/blob/develop/CHANGELOG.md
https://github.com/swagger-api/swagger-ui/releases
https://github.com/Redocly/redoc/blob/main/CHANGELOG.md
https://github.com/NativeScript/nx/blob/main/CHANGELOG.md

I prefer `Features` and `Fixes` because not everyone is familiar with sevmer terminology.

Reply:
Yes, but I mean it's the default format for Changesets and a lot of projects use it, so I don't expect too much troubles with understanding it. Anyways, we already have entries like this, so the changelog needs to be adjusted (and we can do this after the release). I'm going to set up a call to discuss the format we want to use.

Reply:
This format is nice for libraries but most of the "products" out there change the language a bit. Do you have any reasons against using `Features` and `Fixes` (other than there is a little work required on our end)?

Reply:
Not at all, apart it's a default which is always a good idea to stick with.